spec/features/user_can_login_spec.rb

```diff
@@ -25,4 +25,15 @@
     expect(page).to have_current_path("/posts")
     expect(page).to have_content("Welcome to Acebook by D-Railed!")
   end
+
+  scenario "User inputs wrong email and password in details" do
```

Comment:
Could you please change the test to show that the user tries to log in with an invalid email or password?
1. sign up(email, password)
2. sign out
3. sign up(invalid email, password) OR sign up(email, invalid password)

Reply:
Done! Creating another pull request in a minute.